app/enterprise/0.35-x/developer-portal/configuration/authentication/sessions.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Sessions in the Dev Portal
+---
+
+⚠️**Important:** Portal Session Configuration does not apply when using [OpenID Connect](/hub/kong-inc/openid-connect) for Dev Portal authentication. The following information assumes that the Dev Portal is configured with `portal_auth` other than `openid-connect`, for example `key-auth` or `basic-auth`.
+
+## How does the Sessions Plugin work in the Dev Portal?
+
+When a user logs in to the Dev Portal with their credentials, the Sessions Plugin will create a session cookie. The cookie is used for all subsequent requests and is valid to authenticate the user. The session has a limited duration and renews at a configurable interval, which helps prevent an attacker from obtaining and using a stale cookie after the session has ended.
+
+The Session configuration is secure by default, which may [require alteration](#session-security) if using HTTP or different domains for [portal_api_url](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_api_url) and [portal_gui_host](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_gui_host). Even if an attacker were to obtain a stale cookie, it would not benefit them since the cookie is encrypted. The encrypted session data may be stored either in Kong or the cookie itself.
```

Comment:
Do we have a recommendation on where the encrypted data should be stored? (cookie vs Kong)

Reply:
(or explanation on why it must be set to Kong for the Dev Portal)

Reply:
@levfishbluefish we explicitly left that out of Session Plugin doc. IIRC from discussion with @p0pr0ck5 in January when looking at that doc, we simply say what the property is and where things are stored, and leave the decision (and its implications) to the user.